src/Tizen.NUI/src/public/BaseComponents/TextEditor.cs

```diff
@@ -883,6 +883,43 @@ public string InputUnderline
             }
         }
 
+        /// <summary>
+        /// The Strikethrough property.
+        /// </summary>
+        /// <remarks>
+        /// <see cref="Tizen.NUI.Text.Strikethrough"/>
+        /// </remarks>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Strikethrough Strikethrough
+        {
+            get
+            {
+                return (Strikethrough)GetValue(StrikethroughProperty);
+            }
+            set
+            {
+                SetValue(StrikethroughProperty, value);
+                NotifyPropertyChanged();
+            }
+        }
+
+        /// <summary>
+        /// The InputStrikethrough property.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string InputStrikethrough
```

Comment:
What is diffrent `input` or not `input` (`Strikethrough` / `InputStrikethrough`)

Reply:
Hi @myroot , the `InputStrikethrough` is the strikethrough parameters of the new input text. and the `Strikethrough` is the default strikethrough parameters. 
We followed same data-types of (`Underline` / `InputUnderline`).

Please could you check the InputUnderline in the following documentation 
https://developer.tizen.org/development/guides/.net-application/natural-user-interface/ui-components/texteditor

Reply:
Could you explain means of `new input text`?

is it a text that was typed by User throuht IME?

I can't find definition of `InputXXX` on the link that shared by you

Reply:
What is the IME ?

Reply:
another words, on screen keyboard

https://en.m.wikipedia.org/wiki/Input_method

Reply:
Could you explain about `InputXXXX` compare with others platform(android, ios, web...)?

Reply:
I tried to find similar properties on Android and CSS(web).
But nothing is similar to this, maybe the intent was the below behavior in CSS.
Changing style during typing/focus in TextEditor  or TextFiled.
Please try the below code.
```
<!DOCTYPE html>
<html>
<head>
<style>
input:focus {
  background-color: yellow;
  color: green;
}
</style>
</head>
<body>

<p>Click inside the text field to see a yellow background:</p>

<form>
  Test : <input type="text" name="test"><br>
</form>

</body>
</html>
```
Is InputXXX should work during focus OR new added text OR this is an issue and we have to remove InputStrikethrough  ?!
@myroot , @bowon What do you think?

Reply:
@shrouqsabah 
Thanks for research it

I suppose, InputXXXX series was made to apply font attribute as partially.
But it does not works as intended
I also test InputColor.

InputColor was not working when any text are not yet typed, but when any text was typed and deleted it, InputColor was working
![inputtext-error1](https://user-images.githubusercontent.com/1029155/132608717-98cfb18c-ef74-41a5-b3f4-a68e1ed40a03.gif)

and InputColor does not work depending on the situation.
If the input text is more deleted than when you applied InputColor, it does not work.
![inputtext-error2](https://user-images.githubusercontent.com/1029155/132609010-22a2b923-f822-479a-a146-27715ecbb616.gif)

So, InputXXX series API provide a bad API usability, 
I hope to improve these api usability

I also check others platform, 
how to provide way to apply text attribute as partially

in android, they provide spannable text in EditText, it enable to apply text attribute to use span with bound of text

return to here
I think, no need to add a new APIs for InputXXX series such as InputStrikethrough

But, I have not permission to make this decision

Reply:
@myroot , 
Thanks for this clarification. I agree with you.

I think you are talking about `Span tag` in Markup.
We implemented this feature. Please find the patch (merged).
**Add support for span tag** : https://review.tizen.org/gerrit/c/platform/core/uifw/dali-toolkit/+/257786

@wonrst , please could you advise about removing InputStrikethrough.

Reply:
> I tried to find similar properties on Android and CSS(web).
> But nothing is similar to this, maybe the intent was the below behavior in CSS.
> Changing style during typing/focus in TextEditor or TextFiled.
> Please try the below code.
> 
> ```
> <!DOCTYPE html>
> <html>
> <head>
> <style>
> input:focus {
>   background-color: yellow;
>   color: green;
> }
> </style>
> </head>
> <body>
> 
> <p>Click inside the text field to see a yellow background:</p>
> 
> <form>
>   Test : <input type="text" name="test"><br>
> </form>
> 
> </body>
> </html>
> ```
> 
> Is InputXXX should work during focus OR new added text OR this is an issue and we have to remove InputStrikethrough ?!
> @myroot , @bowon What do you think?

@bowon , I am sorry to bother you with this mention. I want to mention Bowon Ryu :)
@wonrst , What do you think?